extensions/wikibase/src/org/openrefine/wikibase/qa/scrutinizers/ItemRequiresScrutinizer.java

```diff
@@ -106,6 +107,9 @@ public void scrutinizeStatementEdit(StatementEntityEdit update) {
                     issue.setProperty("property_entity", propertyId);
                     issue.setProperty("added_property_entity", itemRequiresPid);
                     issue.setProperty("example_entity", update.getEntityId());
+                    if (constraint.itemList.size() == 1) {
+                        issue.setProperty("item_entity", (ItemIdValue) constraint.itemList.get(0));
+                    }
```

Comment:
This is the place where you can also change the warning id that you are emitting.

Reply:
QAWarning did not have setter for type so extended logic in the existing QA warning constructor to set correct warning type